interfaces/default/js/dash.js

```diff
@@ -172,6 +172,33 @@ function loadRecentMovies() {
   })
 }
 
+function loadRecentMoviesList() {
+  if (!$('#latestmovieslist_table_body').length) return
+  $.getJSON(WEBDIR + 'kodi/GetRecentMovies', function(result) {
+    if (result.movies[0] == null) {
+      $('#latestmovieslist_table_body').append(
+        $('<tr>').append($('<td>').html('No latest movies found').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2'))
+      )
+      return
+    }
+    $.each(result.movies, function(i, item) {
+
+      var fanart = WEBDIR + 'kodi/GetThumb?h=240&w=430&thumb=' + encodeURIComponent(item.fanart)

```

Comment:
Hi! I use in https://github.com/LopezRegaladoFj/HTPC-Manager/blob/37af2abb3efbae7c8b4f5aaed7b6fad36d2dbfa2/interfaces/default/html/dash.html lines 314,315,316